src/main/java/org/broadinstitute/hellbender/utils/genotyper/LikelihoodMatrix.java

```diff
@@ -14,13 +14,13 @@
      * List of evidence in the matrix sorted by their index therein.
      * @return never {@code null}.
      */
-    public List<EVIDENCE> evidence();
+    List<EVIDENCE> evidence();
```

Comment:
Do you mean to get rid of the `public`s here?

Reply:
in Interfaces public is implied.

Reply:
Ah, of course!  I forgot it was an interface.